### ISSUE TITLE
Switching between steady and unsteady is not user friendly.

### ISSUE THREAD

**Issue by Paulie14** [flow123d/flow123d#1201]
Problem reported by Milan Hokr.
Problem noted also in #912, but considers only changing documentation.

Dokumentace 3.0.3 pořád píše totéž co 2.2.1, tj. že přepínání se dělá
přes nulovou nebo nenulovou hodnotu storativity.

Nepraktické ale je, když chci jednu úlohu ve verzi ustálené i neustálené
a na to pak musím opravit hodnotu storativit na mnoha místech pokud mám
více domén, riziko chyby. Jeden přepínač by byl lepší - to také v
komentáři zmiňuješ jako plán.

Navíc, chování je nelogické, pokud mám nenulovou storativitu a zároveň  
" end_time: 0"  (můj první pokus o změnu neustálené na ustálenou ve
verzi 2.2.1) - spočte nultý krok, do kterého dá nějakou výchozí hodnotu,
ani ne zadanou počáteční podmínku, ani ne ustálené řešení s danými
okrajovými. Ve 3.0.3 jsem nezkoušel, ale dokumentace je psaná stejně.

**Comment by jbrezmorf** [comment]
Noted in  #912

**Comment by jbrezmorf** [comment]
Diskuse nebyla dotazena.
Issue je stale "open" tedy nevyresene. 
Prepinani je zatim stale pres storativitu. 

Jak by to bylo vhodne?
Mit tam field "time_term" ktery by efektivne prenasoboval casovy clen v rovnici?
Time by se daly mit v ruznych castech oblasti steady a unsteady, byo by to konzistentni se
soucasnym schematem a snad by to i v teto obecnosti mohlo fungovat.

**Comment by Paulie14** [comment]
vzal jsem si test 10/02 a zkusil jsem si tam zmenit dve veci:

- storativity=0: napocitalo to 50x stejny ustaleny problem (okp sedi)

- end_time=0: zadny vypocet, vystup:
pressure_p0 odpovida pocatecni podmince (dal jsem =2)
pressure_p1: nejaka divna interpolace mezi 0.65 - 0.67
velocity_p0: nulova

Zatim nevim, jestli Richards bude delat neco jineho.. Ale toto chovani je zvlastni..

**Comment by milanhokr** [comment]
Ano, chování při end_time odpovídá tomu co v mé úloze - ještě jsem vypsal pressure_p0, to je dle počáteční podmínky, stejně jako piezo_head_p0 a pressure_p1 je nějaké nesmyslné rozložení. Toto chování ale myslím bylo běžné u každé neustálené úlohy v nultém kroku (jen jsem si to pamatoval tak, že p1 bylo přímo nula).

**Comment by jbrezmorf** [comment]
Result:
- review steady/unsteady switching algorithm in DarcyFlow if it could be extended for the case of the time term only on the part of the domain
- even if it is not the case use the field "unsteady" field as a binary time-dependent function for the steady/unsteady switching.

**Comment by jbrezmorf** [comment]
- [ ] Introduce the time_term field with 0, 1 values.
- [ ] Change behavior of Darcy flow.
- [ ] Plan similar behavior for other equations.
